README.md

```diff
@@ -23,16 +25,11 @@ npm install --save-dev raw-loader
 
 <h2 align="center">Usage</h2>
 
-**require**
-```js
-const txt = require("raw!./file.txt")
-```
+Use the loader either via your webpack config or manually.

```

Comment:
'inline', that is the word I was looking for!! That's more clear then 'manually'.